javascript/javascript2/week1/shortword.js

```diff
@@ -0,0 +1,17 @@
+"use strict"
+const danishWords = ["bil", "plante", "kaffe", "bog","ø",  "planetarium"];
+
+function getShortestWord (array){
+    let shortWord = array[0];
+    for(let i=1; i<array.length; i++){     
+        
+       if(array[i].length < shortWord.length){
+           shortWord = array[i];           
+        }
+    }
+    return 'shortest string is '+shortWord;
```

Comment:
it's a bit cleaner to just return `shortWord` and append the string `'shortest string is '` outside of the function